integration-test/src/test/java/com/datastrato/gravitino/integration/test/container/TrinoITContainers.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.container;
+
+import com.datastrato.gravitino.integration.test.util.CommandExecutor;
+import com.datastrato.gravitino.integration.test.util.ProcessData;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.logging.log4j.util.Strings;
+import org.mortbay.log.Log;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableSet;
+
+public class TrinoITContainers implements AutoCloseable {
+  private static final Logger LOG = LoggerFactory.getLogger(TrinoITContainers.class);
+
+  public static String dockerComposeDir;
+
+  private static ImmutableSet<String> servicesName =
+      ImmutableSet.of("trino", "hive_metastore", "hdfs", "mysql", "postgresql");
+
+  Map<String, String> servicesUri = new HashMap<>();
+
+  TrinoITContainers() {
+    String dir = System.getenv("GRAVITINO_ROOT_DIR");
+    if (Strings.isEmpty(dir)) {
+      throw new RuntimeException("GRAVITINO_ROOT_DIR is not set");
+    }
+
+    dockerComposeDir = dir + "/integration-test/trino-it/";
```

Comment:
I think better to change `ITUtils.joinDirPath(dir, "integration-test", "trino-it");`